### PR TITLE
Update project xml parser since data point types are totally changed.

### PR DESCRIPTION
**Pull request by hhou2011**
@Pacane

### CODE REVIEW

File: src/main/java/org/dsa/iot/knx/DeviceFolder.java [hunk 1]
```diff
@@ -92,21 +93,24 @@ protected void addPoint(ActionResult event) {
...
-		Map<String, String> dataPointSubTypeIdToDataPointTypeId = masterDataParser
-				.getDataPointSubTypeIdToDataPointTypeId();
-		if (!dataPointTypeIdToSize.isEmpty() && !dataPointSubTypeIdToDataPointTypeId.isEmpty()) {
-			String contentProject = event.getParameter(ATTR_PROJECT_CONTENT, ValueType.STRING).getString();
-			EtsXmlParser projectParser = new EtsXmlParser(this);
-			projectParser.setDataPointSubTypeIdToDataPointTypeId(dataPointSubTypeIdToDataPointTypeId);
-			projectParser.setDataPointTypeIdToSize(dataPointTypeIdToSize);
+		if (masterDataContent != null && !masterDataContent.isEmpty()) {
+			masterDataParser.parse(masterDataContent);
+
```
Comment (Pacane) [inline src/main/java/org/dsa/iot/knx/DeviceFolder.java]: formatting

File: src/main/java/org/dsa/iot/knx/DeviceFolder.java [hunk 2]
```diff
@@ -92,21 +93,24 @@ protected void addPoint(ActionResult event) {
...
+		}
+	}
+
+	@Override
+	protected void importProjectByXml(ActionResult event) {
+
```
Comment (Pacane) [inline src/main/java/org/dsa/iot/knx/DeviceFolder.java]: formatting

File: src/main/java/org/dsa/iot/knx/masterdata/MasterDataParser.java [hunk 3]
```diff
@@ -56,7 +56,7 @@ public void setDataPointSubTypeIdToDataPointTypeId(Map<String, String> dataPoint
 	}
 
 	public void parse(String content) {
-
+        LOGGER.info("start parsing master data....");
```
Comment (Pacane) [inline src/main/java/org/dsa/iot/knx/masterdata/MasterDataParser.java]: formatting

File: src/main/java/org/dsa/iot/knx/project/OpcFileParser.java [hunk 4]
```diff
@@ -21,7 +21,7 @@ public OpcFileParser(EditableFolder folder) {
 	}
 
 	public void parse(String content) {
-
+        LOGGER.info("start parsing opc file...");
```
Comment (Pacane) [inline src/main/java/org/dsa/iot/knx/project/OpcFileParser.java]: formatting

**Review by Pacane — CHANGES_REQUESTED** [review]

File: src/main/java/org/dsa/iot/knx/project/OpcFileParser.java (hunk 4 above)
Comment (hhou2011) [reply src/main/java/org/dsa/iot/knx/project/OpcFileParser.java]: updated!

**Review by Pacane — APPROVED** [review]

**Pull request merged**